QuartzSandbox/assets/shaders/basic.shader

```diff
@@ -0,0 +1,28 @@
+#shader vertex
+#version 330 core
+
+layout (location = 0) in vec3 aPos;
```

Comment:
Make this ``a_pos`` or make everything else `aCamelCase``, just for consistency.